lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -116,27 +117,41 @@ spec = do
     estimateMaxInputsTests Cardano.Mainnet
     estimateMaxInputsTests (Cardano.Testnet (Cardano.NetworkMagic 0))
 
-    describe "cost of withdrawal" $ do
-        it "one can measure the cost of withdrawal" $ property $ \withdrawal ->
-            let
-                policy :: FeePolicy
-                policy = LinearFee (Quantity 100000) (Quantity 100) (Quantity 0)
+    describe "fee calculations" $ do
+        let policy :: FeePolicy
+            policy = LinearFee (Quantity 100000) (Quantity 100) (Quantity 0)
```

Comment:
Suggestion: Use `NumericLiterals`. (Though I understand this code was not added in this PR.)

```suggestion
            policy = LinearFee (Quantity 100_000) (Quantity 100) (Quantity 0)
```